enterprise/server/backends/authdb/authdb.go

```diff
@@ -113,16 +113,15 @@ func (d *AuthDB) GetAPIKeyGroupFromBasicAuth(ctx context.Context, login, pass st
 		qb.AddWhereClause(`g.write_token = ?`, pass)
 		q, args := qb.Build()
 		existingRow := tx.Raw(q, args...)
-		return existingRow.Scan(akg).Error
+		return existingRow.Take(akg).Error
 	})
 	if err != nil {
 		if db.IsRecordNotFound(err) {
-			return nil, status.UnauthenticatedErrorf("User/Group specified by %s:%s not found", login, pass)
+			return nil, status.UnauthenticatedErrorf("User/Group specified by %s:%s not found", login, redactInvalidToken(pass))
```

Comment:
Let's remove the password entirely from the message. Even with redaction, it can give away enough information.

Reply:
Done